miasm2/ir/ir.py

```diff
@@ -256,15 +258,35 @@ def __init__(self, label, irs):
 
         assert isinstance(label, AsmLabel)
         self.label = label
-        self.irs = irs
-        self.except_automod = True
+        self._assignments = tuple(irs)
         self._dst = None
         self._dst_linenb = None
 
-    def _get_dst(self):
-        """Find the IRDst affectation and update dst, dst_linenb accordingly"""
-        if self._dst is not None:
-            return self._dst
+    @property
+    def irs(self):
+        return self._assignments
+
+    def reset_dst(self):
+        self._dst = None
+        self._dst_linenb = None
+
+    def is_dst_set(self):
+        return self._dst is not None
+
+    def add_assignblock(self, assignblock):
```

Comment:
A "del" or "remove" is missing to complete the API.
Why not having just an API to create a new IRBlock with a list of assignments?

`AssignBlock` doesn't have these kind of API (add, update, del, ...)